holonix/src/test.nix

```diff
@@ -0,0 +1,14 @@
+let
+  pkgs = import ../nixpkgs/nixpkgs.nix;
+
+  name = "hc-test";
+
+  script = pkgs.writeShellScriptBin name
+  ''
+  hc-rust-fmt-check \
+  && hc-qt-c-bindings-test \
+  && hc-rust-wasm-compile \
```

Comment:
looks like we are missing `hc-rust-test` here and also it runs `hc-rust-wasm-compile` so that should be removed here.